source/github-events/on-new-comments.ts

```diff
@@ -7,6 +7,8 @@ const delegates = new Set<delegate.Subscription>();
 const observer = new MutationObserver(run);
 
 function run(): void {
+	console.log(new Date().toLocaleString().split(',').pop()!.trim());
```

Comment:
Logged successful 😄

Reply:
(｢•-•)｢ ︵ ┻━┻

That's what I get for using `git commit --all` every time